main_tracker.py

```diff
@@ -0,0 +1,104 @@
+import os
+import argparse as ap 
+from ultralytics import YOLO
+import glob  
+import pickle 
+
+import cv2 
+
+if __name__ == "__main__":
+    ap = ap.ArgumentParser()
+    ap.add_argument("-p", "--pretrained", default="runs_from_gpu/run_q4/detect/train14/weights/best.pt", help="model to use for training. default = yolov8s")
```

Comment:
Make it as required, but remove the default value?

Reply:
I checked in this code so you or others know what I am doing or can use it.  
Do we really need it to make it general ?